Doc/howto/enum.rst

```diff
@@ -998,11 +998,12 @@ Plain :class:`Enum` classes always evaluate as :data:`True`.
 """""""""""""""""""""""""""""
 
 If you give your enum subclass extra methods, like the `Planet`_
-class below, those methods will show up in a :func:`dir` of the member,
-but not of the class::
+class below, those methods will show up in a :func:`dir` of the member and the
+class. Attributes defined in an :func:`__init__` method will only show up in a
+:func:`dir` of the member::
 
     >>> dir(Planet)
-    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__doc__', '__members__', '__module__']
+    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__doc__', '__init__', '__members__', '__module__', 'surface_gravity']
```

Comment:
Does `__init__` need to be here?  It will never be called after the enum class is created.

Reply:
`surface_gravity` is here so completion works?

Reply:
I think having `__init__` show up on `dir(Planet)` improves the output of `help(Planet.EARTH)`. Without `__init__` being present on `dir(Planet)`, there is no reference in the output from `help(Planet.EARTH)` to the fact that the member has two named attributes, `mass` and `radius`. With `__init__` being present on `dir(Planet)`, however, the signature of the `__init__` function shows up in the output of `help(Planet.EARTH)`, so we can clearly see that the member has two named attributes, `mass` and `radius`.